deployments/jwebbinar/config/common.yaml

```diff
@@ -21,7 +21,7 @@ jupyterhub:
         c.KubeSpawner.lifecycle_hooks = {
             "postStart": {
                 "exec": {
-                    "command": ["/bin/sh", "-c", "rsync -r /etc/default-home-contents/ /home/jovyan; /opt/environments/setup-notebooks;  chmod +x /opt/conda/envs/jwebbinar/bin/ds9; rm -rf /home/jovyan/.local/share/jupyter/kernels/ds9; rm /home/jovyan/Desktop/DS9.desktop; chmod +x /home/jovyan/Desktop/*.desktop; rm -f /home/jovyan/.jdaviz-example-data; rm -f /home/jovyan/Desktop/build.out;  /opt/common-scripts/symlink-crds;"]
```

Comment:
The `rm`s are here because unfortunately, on the JWebbinar platform, existing users may already have these files in their environments.  Not sure how else to handle that problem, but we probably need a better solution as more platforms "go live".

Reply:
One possible alternative is to wait until we're ready to field the new image,  then do a  "find /efs ..." on the CI-node and carefully rm them for all users.   Your call,  don't care either way.

Reply:
That's a great idea.